tests/installation/livecdreboot.pm

```diff
@@ -81,9 +82,10 @@ sub run() {
             save_screenshot;
         }
         if (get_var("XEN")) {
+            send_key 'up';    # prevent grub2 timeout
```

Comment:
even normal boot? it should worked till one day the order of boot entry are changed, but I don't think it will happen :)
fine with me, will update soon.